contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -718,10 +709,21 @@ func (s *SchedulerServer) bootstrap(hks hyperkube.Interface, sc *schedcfg.Config
 		return n.(*api.Node)
 	}
 
-	fcfs := podschedulers.NewFCFSPodScheduler(as, lookupNode)
+	execInfoCache, err := executorinfo.NewCache(defaultExecutorInfoCacheSize)
```

Comment:
panic'ing in constructors is definitely less idiomatic than returning an err :-/